### PR TITLE
Use template file to generate ready to use script.  

### PR DESCRIPTION
**Pull request by NelsonScott**
Get user input for username and password to use with sftp

### CODE REVIEW

File: linux/sample_driver.sh [hunk 1]
```diff
@@ -9,11 +9,11 @@ DIRECTORY="/tmp/csvs"
 
 echo "Directory: $DIRECTORY."
 
-# Loop through every CSV file in the directory and call sendcsv.bat
+# Loop through every CSV file in the directory and call send_csv.sh
 DIRECTORY="$DIRECTORY/*.csv"
 for f in $DIRECTORY; do
 	echo "Uploading: $f."
-	./sendcsv.sh "$f"
+	./send_csv.sh "$f"
```
Comment (ejuher) [inline linux/sample_driver.sh:16]: why change to snake case? it looks like the file is integration/linux/sendcsv.sh

Comment (ejuher) [inline linux/sample_driver.sh:16]: ah, i think i understand that generate_script.rb creates a send_csv.sh file. How do they run generate_script? should you delete the sendcsv.sh file thats currently in the repo?

Comment (NelsonScott) [inline linux/sample_driver.sh:16]: Yeah exactly send_csv would be generated by us.  I think it's a lot to ask clients to generate their own SSH keys and then edit these scripts to include the right usernames and passwords.  With this PR we do that from our end and just send them send_csv.sh

File: linux/README.txt [hunk 2]
```diff
@@ -1,82 +0,0 @@
...
-   the end of the script. Lots of comments have been provided to assist.
-
-   Example: sendcsv.sh myfile.csv
-
-You can also integrate the sendcsv.sh into your existing processes. For an
-example of using sendcsv.sh, see sample_driver.sh. sample_driver.sh will
-upload an entire directory full of CSVs. To change the directory, open
-sample_driver.sh, and edit the DIRECTORY parameter.
-   
-============================ Generating an SSH Key ============================
```
Comment (ejuher) [inline linux/README.txt:42]: should these instructions be replaced somewhere else? are are we no longer going to use ssh?

File: linux/sample_driver.sh (hunk 1 above)
Comment (NelsonScott) [inline linux/sample_driver.sh:16]: sendcsv -> sendcsv_template.sh

File: linux/README.txt (hunk 2 above)
Comment (NelsonScott) [inline linux/README.txt]: checkout linux/README.md

**Comment by ejuher** [comment]
lgtm

**Pull request merged**